test/component-bind.spec.ts

```diff
@@ -299,42 +289,12 @@ describe('component', () => {
       widget.append(child);
 
       expect(() => child.text = 23).to.throw(
-        'Binding "myText" <-> "#textInput1.text" failed to update CustomChild property "text": '
-        + 'Expected value "23" to be of type string, but found number.'
-      );
-    });
-
-    it('throws if target value has changed to wrong type', () => {
-      class CustomChild extends Composite {
-        private _text: string | number;
-        get text(): string | number {
-          return this._text;
-        }
-        set text(value: string | number) {
-          this._text = value;
-        }
-      }
-      const child = new CustomChild({id: 'textInput1'});
-      child.text = 'foo';
-      widget.append(child);
-
-      child.text = 23;
-
-      expect(() => widget.myText).to.throw(
-        'Binding "myText" <-> "#textInput1.text" failed to provide CustomComponent property "myText": '
```

Comment:
Why did we have this getter check before?

Reply:
as mentioned above